myia/compile/transform.py

```diff
@@ -79,6 +82,43 @@ def wrap_primitives(graph):
     return graph
 
 
+def gather_handles_params(params):
+    """Return list of all handles in params.
+
+    This list also has graph nodes that correspond to the handle value
+    and functions to extract the value from the arguments.
+    """
```

Comment:
That function doesn't seem to do...  much.

Reply:
It did more in the beginning and might go back to doing more if I ever add back the support that was cut to get this working.

Unless you would rather we cut the whole thing for now.

Reply:
I mean that `gather_handles_params` doesn't do anything at all. It's a docstring and then no function body.

Reply:
Oh right.  This is more leftovers then.